seed-job/template.yaml

```diff
@@ -110,7 +110,11 @@ objects:
                         }
                         stage('Build the container image') {
                             dir("${PIPELINE_NAME}/${GIT_PATH}"){
-                                sh "docker build --no-cache -t ${image_name} -f ${TARGET_FILE} ${BUILD_CONTEXT}"
+                                echo "Cleanup the local parent image if existing"
+                                def output = sh(returnStdout:true, script: "docker rmi $(cat ${TARGET_FILE}|grep '^FROM'|awk '{print $2}') > image-remove 2>&1", returnStatus: true)
+                                sh "cat image-remove"
+                                echo "Build the image by always pulling the base image"
```

Comment:
This bit is not required IMO.

Reply:
This is to make sure, we are not creating dangling images.

Reply:
`grep '^FROM'|awk '{print $2}')` could be replaced by `awk /FROM/ {print $2}` thus resulting in only one pipe instead of two.

Reply:
```
def base_image = sh(returnStdout:true, script: "cat ${TARGET_FILE} | awk '/FROM/ {print $2}'", returnStatus: true)
def base_image_id = sh(returnStdout:true, script: "docker images -q ${base_image}")
docker build --pull [..]
def base_image_id2 = sh(returnStdout:true, script: "docker images -q ${base_image}")
sh(returnStdout:true, script: "docker rmi ${base_image_id} ${base_image_id2}")
```